udata_front/views/dataset.py

```diff
@@ -78,7 +78,7 @@ def can(self, *args, **kwargs):
 @blueprint.route('/<dataset:dataset>/', endpoint='show')
 class DatasetDetailView(DatasetView, DetailView):
     template_name = 'dataset/display.html'
-    page_size = 4
+    page_size = 8
```

Comment:
We could rename this to explicit object as well here:
```suggestion
    reuse_page_size = 8
```